x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvAssert.java

```diff
@@ -145,6 +146,10 @@ private static void assertMetadata(
                     // Type.asType translates all bytes references into keywords
                     continue;
                 }
+                if (blockType == Type.DOUBLE && expectedType == DENSE_VECTOR) {
+                    // DENSE_VECTOR is internally represented as a double block
```

Comment:
This could potentially change when we support byte and bit element types - we could create the appropriate blocks.